skills/csharp/experimental/automotiveskill/AutomotiveSkill.csproj

```diff
@@ -14,25 +14,14 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.0" />
     <PackageReference Include="Microsoft.Azure.CognitiveServices.ContentModerator" Version="2.0.0" />
     <PackageReference Include="Microsoft.Azure.CognitiveServices.Language" Version="1.0.1-preview" />
-    <PackageReference Include="Microsoft.Bot.Builder" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.8.1" />
     <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.8.1-rc0" />
-    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.8.1" />
-    <PackageReference Include="Microsoft.Bot.Schema" Version="4.8.1" />
     <PackageReference Include="Microsoft.Bot.Solutions" Version="0.9.0-daily252" />
-    <PackageReference Include="Microsoft.Graph" Version="1.15.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="SharpYaml" Version="1.6.5" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="all" />
```

Comment:
can we move this to the Directory.Build.Props file so we can remove this from all skills?

Reply:
Good point. we may open another issue to track moving all projects shared properties to this file.